lib/MetaCPAN/Web/Model/ReleaseInfo.pm

```diff
@@ -133,37 +133,54 @@ sub normalize {
     sub {
         my $data = shift;
 
-        Future->done( {
-            took => max(
-                grep defined,
-                map $_->{took},
-                grep is_hashref($_),
-                values %$data
-            ),
-            notification => $data->{notification}{notification},
-            coverage     => $data->{coverage}{coverage},
-            release      => $data->{release}{release},
-            favorites    => $data->{favorites}{favorites},
-            rating       => $data->{rating}{rating},
-            versions     => $data->{versions}{versions},
-            distribution => $data->{distribution}{distribution},
-            author       => $data->{author}{author},
-            contributors => $data->{contributors}{contributors},
-            irc          => $self->groom_irc( $data->{release}{release} ),
-            issues       => $self->normalize_issues(
-                $data->{release}{release},
-                $data->{distribution}{distribution}
-            ),
-            plussers => $data->{plussers}{plussers},
-            (
-                $self->full_details
-                ? (
-                    files   => $data->{files}{files},
-                    modules => $data->{modules}{modules},
-                    changes => $data->{changes}{changes},
-                    )
-                : ()
-            ),
+        my $n_data = $data->{notification};
+        my $n_future;
+        if ( $n_data->{notification} ) {
+            $n_future
+                = $self->_author->get_multiple( @{ $n_data->{notifiable} } )
```

Comment:
Any logic like this should move into the API::Permissions get_notification_type method.

Reply:
Now implemented, and ready for re-review!